<launch>

   <!-- Serial Node to CRIO -->
   <node pkg="rosserial_python" type="serial_node.py" name="SIRL" args="/dev/ttyUSB0"/>

</launch>
